tasks/config.ts

```diff
@@ -155,24 +156,46 @@ export function load() {
     });
   });
 
-  async function getPriceFeedsConfig(hre: HardhatRuntimeEnvironment, deployer: SignerWithAddress) {
-    const priceFeeds = Object.entries(PriceFeedContractNames).map(async ([key, value]) => {
-      const instance = await hre.ethers.getContract(value, deployer);
-      return { symbol: key, address: instance.address, decimals: await getDecimals(instance) };
+  async function getPriceFeedsConfig(hre: HardhatRuntimeEnvironment, deployer: SignerWithAddress): Promise<{ [key: string]: PriceFeed }> {
+    // TODO - Multichain: This approach won't work when we're not the ones deploying the contract (for example when Chainlink is the one doing it) - https://lagoslabs.atlassian.net/browse/LAGO-890
+    const priceFeedsPromises = Object.entries(PriceFeedContractNames).map(async ([key, value]) => {
+      const priceFeedContract = await hre.ethers.getContract(value, deployer);
+      const priceFeedData: PriceFeed = {
+        address: priceFeedContract.address,
+        decimals: await getPriceFeedDecimals(priceFeedContract),
+        updatable: isPriceFeedUpdatable(priceFeedContract)
+      };
+
+      return { [key]: priceFeedData };
     });
 
-    const results = await Promise.all(priceFeeds);
-    return results;
+    const results = await Promise.all(priceFeedsPromises);
+
+    return results.reduce((previous, current) => {
+      const key = Object.keys(current)[0];
+      previous[key] = current[key];
+
+      return previous;
+    }, {});
```

Comment:
it's a little difficult to understand what exactly happens in lines 174-179
instead, you can use async reduce in line 161 and build the `{ [key: string]: PriceFeed }` object that will be returned from this function